plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -158,7 +158,7 @@ protected TestTable createTableWithUnsupportedColumn()
     {
         return new TestTable(
                 onRemoteDatabase(),
-                "test_unsupported_column_present",
+                "tpch.test_unsupported_column_present",
```

Comment:
related?

Reply:
Yes. Majority of overrides include the schema name. If I didn't add it here, I'd have to override the method that uses this test table.

Reply:
shouldn't this be `getsession().getschema().orthrow()`?

Reply:
Other implementations also put `tpch` explicitly.